release.sh

```diff
@@ -68,6 +70,22 @@ compare_and_release() {
   fi
 
   if [ -n "$NEXT" ] && [ "$NEXT" != "false" ]; then
+    changelog
+
+    git config --global user.email "travis@travis-ci.org"
+    git config --global user.name "Travis CI"
+
+    git add CHANGELOG.md
+    git commit -v --edit -m "${NEXT} Release :tada: :fireworks: :bell:" -m "[skip ci]"
```

Comment:
@samirsilwal I used this. Much clean, such wow! 🙏 

https://stackoverflow.com/a/5070502/2352587